app/konnect/getting-started/configure-runtime.md

```diff
@@ -22,12 +22,15 @@ runtime instances.
 
 ## Prerequisites
 
-* You have a {{site.konnect_product_name}} account. Contact your sales
-representative for access.
+* You have **Runtime Admin** or **Organization Admin** permissions in
+{{site.konnect_saas}}. If you created this account, the account has
+organization admin permissions by default.
 * Tools and permissions:
   * **All platforms:** [Docker](https://docs.docker.com/get-docker/) and [jq](https://stedolan.github.io/jq/) installed
   * **Linux:** User added to the [`docker` group](https://docs.docker.com/engine/install/linux-postinstall/)
-  * **Windows:** Docker Desktop [installed](https://docs.docker.com/docker-for-windows/install/#install-docker-desktop-on-windows) and [integrated with a WSL 2 backend](https://docs.docker.com/docker-for-windows/wsl/)
+  * **Windows:** The quick setup script does not currently work on Windows.
+  See the [Advanced](/konnect/runtime-manager/gateway-runtime-docker/#advanced-setup)
+  instructions instead, then move on to [configuring a service](/konnect/getting-started/configure-service).
```

Comment:
The quick script does work though if you are in a WSL2 backend (e.g the installed Ubuntu); that was the intended audience when @artomal and I tested things out back in the day.

Reply:
Ah yeah, I definitely misunderstood then - I thought the script was just not working at all, prereq or no. That's way better then.

Reply:
Yeah I think the confusion might be that it wasn't called out we should be using a bash shell. Documentation is so hard and you and your team really don't get enough thanks; so THANK YOU!!!

Reply:
And that's a good point too - why do they need WSL 2? Because bash. If they install that but not think to use bash (I mean, maybe, possible), that could throw them off too.

Reply:
100% agree, I think at the time we were just thinking of the easiest thing and keep the instructions the same; that is why we just tested things using bash. In hindsight that may have not the best approach.